### PR TITLE
Feature 5512

### PR DESCRIPTION
**Pull request by vineelsonnathi**
### Redmine ticket(s)
* https://devops.dchbx.org/redmine/issues/7471
* https://devops.dchbx.org/redmine/issues/5512
* https://devops.dchbx.org/redmine/issues/5306

### Local build result

```
bundle exec rake parallel:spec[4]
4011 examples, 0 failures, 78 pendings
Took 155 seconds (2:35)

bundle exec cucumber
56 scenarios (56 passed)
988 steps (988 passed)
6m24.878s

```

### Latest rebase/merge tag
* 1.8.31

---

### Data ticket(s) Followup
* (redmine links here - optional)

### Related Pull Requests
* (github links here - optional)

---

### Peer Review results
```
Finished in 2 minutes 39.8 seconds (files took 7.94 seconds to load)
1183 examples, 0 failures, 16 pending
3911 examples, 0 failures, 77 pendings

bundle exec cucumber
55 scenarios (55 passed)
970 steps (970 passed)
6m33.214s
```
#### Functional Testing
* (For testing locally)

#### Deployment
* (for release manager and/or build manager)

### CODE REVIEW

**Comment by davidplappert** [comment]
@vineelsonnathi Please fix the merge conflicts and the failing build status. Thanks!

File: app/controllers/exchanges/hbx_profiles_controller.rb [hunk 1]
```diff
@@ -250,12 +262,45 @@ def issuer_index
   end
 
   def verification_index
+    @families = Family.by_enrollment_individual_market.where(:'households.hbx_enrollments.aasm_state' => "enrolled_contingent").page(params[:page]).per(15)
```
Comment (Kevin-Wei) [inline app/controllers/exchanges/hbx_profiles_controller.rb]: @lisyk do we still need this?

File: app/views/exchanges/hbx_profiles/_filtered_employers.json.erb [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+    <%= eligibility_criteria(employer) %>
+  <% end %>
+  <%= raw rule.to_json %>,
+  "<%= json_escape format_time_display(employer.xml_transmitted_timestamp) %>",
+  <% transmit_xml_button = capture do %>
+    <%= link_to('Transmit XML', transmit_group_xml_exchanges_hbx_profile_path(employer), method: :post, class: "btn btn-primary transmit-group-xml" , data: { confirm: group_xml_transmitted_message(employer)  }, disabled: make_binder_checkbox_disabled(employer) || employer.can_transmit_xml? ) %>
+  <% end %>
+  <%= raw transmit_xml_button.to_json %>
+  ]
+<% end %>
```
Comment (davidplappert) [inline app/views/exchanges/hbx_profiles/_filtered_employers.json.erb]: This seems like a lot of Ruby code for a view. Maybe this should be moved in the controller?

File: spec/factories/relationship_benefits.rb [hunk 3]
```diff
@@ -1,7 +1,7 @@
 FactoryGirl.define do
   factory :relationship_benefit do
     relationship     :employee
-    premium_pct             40
+    premium_pct             55
```
Comment (davidplappert) [inline spec/factories/relationship_benefits.rb]: This seems like a random fix. Can you verify that this was supposed to be updated?

Comment (Kevin-Wei) [inline spec/factories/relationship_benefits.rb]: Not sure how it even passed before as employee's relationship benefit must be over 50%

**Pull request merged**